src/bundle/Resources/public/js/scripts/core/tag.view.select.js

```diff
@@ -88,11 +88,12 @@
         }
 
         removeItems(items) {
```

Comment:
It would be good to change `items` to `itemsIds` in this PR as now it wrongly suggests that we have whole items' objects under it.
```suggestion
        removeItems(itemsIds) {
```